packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -143,11 +143,24 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
 
     if (file) {
       const reader = new FileReader();
-
+      const emailRegex = /^([A-Z0-9_+-]+\.?)*[A-Z0-9_+-]@([A-Z0-9][A-Z0-9-]*\.)+[A-Z]{2,}$/i;
       reader.onload = (e) => {
         const contents = e?.target?.result as string;
-        const values = contents?.split(",").map((email) => email.trim().toLocaleLowerCase());
-        newMemberFormMethods.setValue("emailOrUsername", values);
+        const lines = contents.split("\n");
+        const validEmails = [];
+        for (const line of lines) {
+          const columns = line.split(/,|;|\|| /);
+          for (const column of columns) {
+            const email = column.trim().toLowerCase();
+
+            if (emailRegex.test(email)) {
```

Comment:
i wonder if we should just z.email(email).parse() for this to keep inline with the rest of our app testing (They do pretty much the same thing just a NIT)